classes/persistent/period.php

```diff
@@ -25,6 +25,7 @@
 namespace mod_cpdlogbook\persistent;
 
 use coding_exception;
+use tool_customlang\local\mlang\langstring;
```

Comment:
this doesn't seem right, what this needed for? Generally speaking plugins shouldn't have dependencies on stuff inside other plugins (or for exceptions when they really do that should be declared in the version.php file)

lang_string is declared in moodlelib, vs langstring looks unrelated

Reply:
PHPStorm must've automatically put that `use` there. I'm removed it now.